src/layouts/LatestVersions.astro

```diff
@@ -0,0 +1,142 @@
+---
+import Page from "./Page.astro";
+
+const props = Astro.props;
+---
+<Page content={props.content} frontmatter={props.frontmatter}>
+    <noscript>
+        <article class="message is-danger">
+            <div class="message-body">
+                <p>Javascript is required to view this page</p>
+            </div>
+        </article>
+    </noscript>
+
+    <p>
+        This page provides the latest versions of Quilt and its components, ready to copy paste into your <code>libs.versions.toml</code> file.
+        It is located in the <code>gradle</code> folder if you are using our template mod.
+    </p>
+
+    <div class="is-flex is-flex-direction-row">
+        <div class="select">
+            <select id="version-select">
+            </select>
+        </div>
+        <div class="field pl-2">
+            <p class="control has-icons-left">
+                <input id="version-search" class="input" placeholder="Search" />
+                <span class="icon is-small is-left">
+                    <i class="fas fa-magnifying-glass"></i>
+                </span>
+            </p>
+        </div>
+    </div>
+
+    <pre id="code-block" class="mt-4">
+# Loading data, please wait...
+</pre>
+
+    <article id="unavailability-warning" class="message is-danger is-hidden">
+        <div class="message-body">
+            <p><strong>Warning:</strong> Quilt Mappings and/or Quilted Fabric API is currently unavailable for this version.</p>
+        </div>
+    </article>
+
+    <button id="copy-button" class="button">
+        <span class="icon">
+            <i class="fas fa-copy"></i>
+        </span>
+        <span>Copy to Clipboard</span>
+    </button>
+</Page>
+
+<script>
+    import "prismjs";
```

Comment:
Could we use the same PrismJS configuration as [the developer wiki](https://github.com/QuiltMC/developer-wiki/blob/2c9728db1adf9ffab626f36adabc704f76a150d8/src/lib/HtmlHead.svelte), for consistency and light mode support ?

Reply:
Light support was added in 0707dd4